build.gradle

```diff
@@ -82,6 +99,11 @@ allprojects {
 
   jacoco { toolVersion = '0.8.2' }
 
+  task sourcesJar(type: Jar, dependsOn: classes) {
+    classifier = 'sources'
+    from sourceSets.main.allSource
+  }
+
```

Comment:
Do we want the java doc jars as well?

Reply:
Do we have any meaningful java doc? If yes, I think we should do it.

Reply:
The javadocs isn't great, very little docs if any I could see